lib/wallet/src/Cardano/Wallet/Write/Tx.hs

```diff
@@ -543,3 +736,12 @@ withStandardCryptoConstraint
 withStandardCryptoConstraint era a = case era of
     RecentEraBabbage -> a
     RecentEraAlonzo -> a
+
+withCLIConstraint
+    :: RecentEra era
+    -> (CLI (ShelleyLedgerEra era) => a)
```

Comment:
No need to wrap this, AFAICT:

```suggestion
withCLIConstraint :: RecentEra era -> (CLI (ShelleyLedgerEra era) => a)
```

Reply:
Keeping it for similarity with `withStandardCryptoConstraint`.